pkg/runctl/run_manager_test.go

```diff
@@ -293,9 +317,9 @@ func Test_RunManager_Log_Elasticsearch(t *testing.T) {
 	}
 }
 
-func preparePredefinedSecrets(mockSecretProvider *mocks.MockSecretProvider) {
-	if scmCloneSecretName != "" {
-		mockSecretProvider.EXPECT().GetSecret(scmCloneSecretName).Return(&v1.Secret{}, nil)
+func preparePredefinedSecrets(mockSecretProvider *mocks.MockSecretProvider, scmCloneSecretName ...string) {
+	if len(scmCloneSecretName) == 1 {
+		mockSecretProvider.EXPECT().GetSecret(scmCloneSecretName[0]).Return(&v1.Secret{}, nil)
 	}
```

Comment:
Why does the function expect multiple `scmCloneSecretName`s but uses them only if only one is specified?

Reply:
Changed in later commit